src/components/MelodyDetailsPage.js

```diff
@@ -0,0 +1,120 @@
+import React from "react";
+import ReactAbcjs from "./react-abcjs.js";
+import MusicInputFormContainer from "./MusicInputFormContainer";
+import "./MelodyDetailsPage.css";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faPlayCircle } from "@fortawesome/free-solid-svg-icons";
+
+function MelodyDetailsPage(props) {
```

Comment:
I suggest using destructuring
```suggestion
function MelodyDetailsPage({ melody, user, stats, ... }) {
```